src/app/components/summary/summary.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import {MatSnackBar} from "@angular/material";
+import {RoomService} from "../../services/room.service";
+import {ActivatedRoute} from "@angular/router";
+
+@Component({
+  selector: 'app-summary',
+  templateUrl: './summary.component.html',
+  styleUrls: ['./summary.component.css']
+})
+export class SummaryComponent implements OnInit {
+
+  roomId:string;
```

Comment:
Super minor thing - lack of space. In order to keep the project code's style consistent I highly recommend using some linters. The most popular is ESLint. I use it myself - you can also add it to CI checks, so you won't be able to merge any code that thas not meet the styleguide rules that you have set